arangod/Agency/Agent.cpp

```diff
@@ -1967,6 +1969,18 @@ void Agent::emptyCbTrashBin() {
 
   LOG_TOPIC("12ad3", DEBUG, Logger::AGENCY) << "unobserving: " << envelope->toJson();
 
+  auto* scheduler = SchedulerFeature::SCHEDULER;
+
+  
+  if (scheduler != nullptr) {
+    scheduler->queue(RequestLane::INTERNAL_LOW, [envelope = std::move(envelope)] {
```

Comment:
please check the return value here.
the `queue` function is marked with `ADB_WARN_UNUSED_RESULT`, which means the caller is expect to handle the result (somehow).

Reply:
You still need to process the return value, as otherwise this will make the compiler produce a warning, which makes the build fail on some platforms.

Reply:
addressed in comments

Reply:
Comments will not help against compile warnings.